src/utils/browser.js

```diff
@@ -7,8 +7,13 @@ const win = typeof window !== "undefined" && window.Math === Math ? window : typ
 /* eslint-enable no-new-func, no-nested-ternary */
 
 const doc = win.document;
+const userAgent = win.navigator.userAgent;
+const IS_SAFARI_ON_DESKTOP = userAgent.indexOf("Safari") !== -1 &&
+														userAgent.indexOf("Chrome") === -1 &&
```

Comment:
@jongmoon What's wrong with the indent?

Reply:
Tab has 8 spaces on Github but on my IDE(VS Code), Tab has 2 spaces.
These makes difference.